alert-policies/sonarqube/ActiveDatabaseConnections.yml

```diff
@@ -3,14 +3,14 @@ name: Active Database Connections
 
 # Description and details
 description: |+
-  This alert is triggered if the database connections are below 1 for 5 minutes.
+  This alert is triggered if the database connections are below 2 for 5 minutes.
```

Comment:
@pkudikyala Need clarity on this alert, because below 2 connections means what?
I think we need to mention like max connections or above 10 connections?

Reply:
Hi @mdumpati, Updated the alert query. Kindly review it.
Thanks.